index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>Portfolio</title>
+</head>
+<body>
+  <section class="first-page">
+    <div class="main">
+      <nav class="navbar">
+        <div class="logo"><a href="#">Eros</a></div>
+        <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
+    </nav>
+    <div class="headline">
+    <div class="description">
+      <h1 class="name"> I’m Aime
+          Glad to see you!</h1>
+         
+      <p class="text"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+           </p>
+      </div>
+      <div class="contact">
+          <h1 class="media">LET'S CONNECT:</h1>
```

Comment:
- [ ] I suggest you wrap your <h1 class="media">LET'S CONNECT:</h1> in a button tag or an anchor tag  as it was reauire by the previous reviewer [here](https://github.com/Hazard10-eng/Portfolio/pull/3/files#r1035198443) because it must be a clickable link.

```
<button>
  <h2 class="media">LET'S CONNECT:</h2>
</button>
```
If you noticed, I changed the h1 to h2, because we can't have more than one h1 tags in a website. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info